addOns/help/src/main/javahelp/contents/ui/dialogs/options/certificate.html

```diff
@@ -27,6 +30,21 @@ <H1>Options Client Certificate screen</H1>
 <code>-config certificate.persist=true</code> is only necessary if you would like the settings persisted in ZAP's regular 
 configuration file (so that they apply for subsequent ZAP use).
 </p>
+<h2>Using PKCS11</h2>
+<p>PKCS11 is one of the Public-Key Cryptography Standards. Usually PKCS11 is used with Hardware Tokens (mostly USB style) 
+that store the cryptographic material in a sacure way. Thus usually a driver is needed. The drivers are most commonly provided 
+by the Issuer of the Certificate stored on the token or the provider of the token. Certain tokens (especially if running 
+under Linux) are supported by opensc/libccid combination.</p>
+<p>
+It is possible to set the Client Certificate from the command line with the following switches:
+</p>
+<pre>
+-config certificate.use=true
+-config certificate.experimentalSlotListIndex=true
+</pre>
+</p>Setting PKCS11 Driver, Pin and active certificate is not possible from the command line for the moment.
```

Comment:
```suggestion
</p>Setting PKCS11 Driver, Pin, and active certificate is not possible from the command line for the moment.
```

Reply:
According to https://www.grammarly.com/blog/comma-before-and/ both with and without comma are correct but at least to me it looks cleaner without. I am comming from a country where such usage is considered wrong thus written so.

Reply:
This is about project guidelines though.

Reply:
Then oxford comma it is (also squashed everything in a single commit)